rfcs/convergence/internationalization.md

```diff
@@ -0,0 +1,217 @@
+# RFC: Internationalization patterns
+
+---
+
+_List contributors to the proposal:_ @smhigley
+
+## Problem Statement:
+
+In v8, the custom strings authors must define to use components are all over the place, and have no standard pattern. E.g. Datepicker uses a `strings` prop and imports defaults from an in-package module
```

Comment:
is there a requirement for v9 to ship with default english strings ? it will not remove the requirement for products to define all these strings again, since they will need to do the translation through their pipelines (e.g. Touchdown)

Reply:
The sentence about DatePicker is slightly inaccurate. In v7 it was required to specify strings (with an English defaults object available). In v8 English strings are provided by default, though in the examples we still demonstrate importing and setting the strings to emphasize that localization is required.

Overall in v8, for the string-heavy components (DatePicker, Calendar, ColorPicker) we provide English defaults, possibly to reduce initial friction/potential confusion for people using them. For other components we don't provide default strings. I don't think we have a hard requirement either way for v9.